src/attr/_version.pyi

```diff
@@ -0,0 +1,5 @@
+class VersionInfo:
+    year: int
```

Comment:
I I was being pedantic I would mention that since the class is frozen=True this should be:
```
class VersionInfo:
    @property
    def year(self) -> int: ...
    @property
    def minor(self) -> int: ...
    @property
    def micro(self) -> int: ...
    @property
    def releaselevel(self) -> str: ...
```